apps/src/code-studio/levels/postOnContinue.js

```diff
@@ -38,23 +48,16 @@ export function postMilestoneForPageLoad() {
     level: appOptions.dialog.level,
     result: result.result,
     pass: true,
-    testResult: TestResults.ALL_PASS
-  });
-}
+    testResult: TestResults.ALL_PASS,
+    onComplete: function() {
```

Comment:
It looks like the callback takes a response as a param (https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/reporting.js#L421), if it's possible to leverage that you could remove line 53

Reply:
Good catch, but if it's ok with you, I'd like to keep this as-is.  I agree that taking the response as a param is cleaner, but my intent here was to minimize risk by just moving the code that used to be in `onContinue()` below up here.